server.go

```diff
@@ -1240,6 +1244,34 @@ func newServer(cfg *Config, listenAddrs []net.Addr, chanDB *channeldb.DB,
 		})
 	}
 
+	var healthChecks []*healthcheck.Observation
+	if !cfg.HealthChecks.ChainCheck.Disabled {
+		backendCheck := func() error {
+			_, _, err := cc.chainIO.GetBestBlock()
```

Comment:
Need to double check how this degrades for each backend...I think for `btcd` this might just block indefinitely rather than actually returning immediately with an error.